release_notes/ocp_3_10_release_notes.adoc

```diff
@@ -414,8 +495,6 @@ inspect them using normal `oc` CLI commands.
 move to RPMs or the kubelet system containers (by setting
 `openshift_use_system_containers=true`), to start or run the platform.
```

Comment:
`openshift_use_system_containers` is removed, the transition will be done automatically per the kind of operation during the upgrade.

Reply:
@ganhuang Thanks for your review. This is addressed in https://github.com/openshift/openshift-docs/pull/11106